test/helper.rb

```diff
@@ -48,6 +48,7 @@ def teardown_db
 
 require "shared"
 
+# require 'logger'
```

Comment:
For some reason `logger` wasn't being loaded when I was running the tests locally with Appraisal and the Rails 3.2 gemset. Sometimes it's handy to be able to see the SQL logs when debugging the tests so I've left this here commented out to save searching for the solution again in the future.